internal/pkg/cli/svc_init.go

```diff
@@ -263,6 +264,19 @@ func (o *initSvcOpts) Validate() error {
 			return err
 		}
 	}
+	if len(o.sourcePaths) != 0 {
+		if o.wkldType != manifestinfo.StaticSiteType {
+			return fmt.Errorf("'--%s' must be specified with '--%s %s'", sourcesFlag, typeFlag, manifestinfo.StaticSiteType)
+		}
+		if err := validateStaticSiteSources(o.fs, o.sourcePaths); err != nil {
+			return err
+		}
```

Comment:
is validating a bit redundant? like the error coming from `convertStringsToAssets()` will be pretty similar if a file doesn't exist, right?